packages/terra-dialog/src/Dialog.jsx

```diff
@@ -40,7 +40,11 @@ const Dialog = ({ header, footer, onClose, children, ...customProps }) => {
     customProps.className,
   ]);
 
-  const closeButton = onClose ? <div className={cx('dialog-header-close')}><Button variant="utility" onClick={onClose} isIconOnly icon={<span className={cx('close-icon')} />} /></div> : null;
+  const closeButton = onClose ? (
+    <div className={cx('dialog-header-close')}>
+      <Button variant="utility" onClick={onClose} isIconOnly icon={<span className={cx('close-icon')} aria-label="Close" />} />
```

Comment:
To add the aria-label close, we would need to add internationalization for this component and translations in each locale.

Reply:
closing this PR for now I thought it would be an easy change but looks like its more involved. Currently we do not have planned capacity for this work :( but I can look it into on side.

@emilyrohrbough @bjankord could you point me to existing components or examples/docs which I can refer to on how to add internationalization and translations in each locale.

Reply:
[Terra-alert](https://github.com/cerner/terra-core/tree/master/packages/terra-alert) is a good example of a component using translations. In the component, we ensure the [intl context is defined](https://github.com/cerner/terra-core/blob/master/packages/terra-alert/src/Alert.jsx#L98).  Then we use the [intl format message function on the component's message key)[https://github.com/cerner/terra-core/blob/master/packages/terra-alert/src/Alert.jsx#L121] to correctly pull in the translation. Once all translations are formated, we [export the intl context](https://github.com/cerner/terra-core/blob/master/packages/terra-alert/src/Alert.jsx#L204).

As for translation setup, we add translations to locale-named json files that contain key-value pairs where the component-name-spaced message name is the key and the translated message is the value. See terra-alert's [translations](https://github.com/cerner/terra-core/tree/master/packages/terra-alert/translations). Note: translations must be included for all of the [terra-supported locales](https://github.com/cerner/terra-core/blob/master/packages/terra-i18n/src/i18nSupportedLocales.js).

Reply:
Translations for 'close' have already been done for `terra-application-utility`. We can strip out them out [here](https://github.com/cerner/terra-framework/tree/master/packages/terra-application-utility/translations). fi-FI is currently missing, but will be added soon.